test/hmactest.c

```diff
@@ -214,6 +214,29 @@ static int test_hmac_single_shot(void)
     return 1;
 }
 
+/* https://github.com/openssl/openssl/issues/13210 */
+static int test_hmac_benmocha(void)
```

Comment:
Something like test_hmac_final_update_fail() might be a better name.. (it is unusual to see a authors name in the test).